lib/api/controllers/cli/data.js

```diff
@@ -51,9 +73,9 @@ function data (request) {
     });
   }
 
-  return Promise.all(promises)
+  return Bluebird.all(promises)
     .catch(error => {
-      var kuzzleError = new InternalError(error.message);
+      const kuzzleError = new KuzzleInternalError(error.message);
       kuzzleError.stack = error.stack;
       _kuzzle.pluginsManager.trigger('log:error', '!! An error occurred during the process.\nHere is the original error message:\n' + error.message);
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Split this 149 characters long line (which is greater than 80 authorized). [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonarqube.kaliop.net/coding_rules#rule_key=javascript%3ALineLength)